external/src/main/resources/application.properties

```diff
@@ -22,12 +18,8 @@ quarkus.http.access-log.enabled=true
 quarkus.http.access-log.category=access_log
 quarkus.http.access-log.pattern=combined
 
-
-# For dev only
-#external.org.hawkular.alerts.ui.path=/home/michael/projects/hawkular-alerts-ui
-
 # Kafka bootstrap applies to all topics
-kafka.bootstrap.servers=localhost:9092
+kafka.bootstrap.servers=localhost:29092
```

Comment:
why do we need to change the port as well?

Reply:
Fixed.